ntbs-service/wwwroot/css/notification.scss

```diff
@@ -36,3 +17,99 @@
 .delete-reason-text-area-container {
     width: 750px;
 }
+
+.manage-notification {
+    border-top-color: $phe-red;
+    border-top-width: 10px;
+    border-top-style: solid;
+    padding-top: 16px;
+
+    &.manage-notification--draft {
+        border-top-color: $ntbs-draft;
+    }
+
+    // We're overriding some of the nhs-uk expander styling here to achieve the desired effect.
+    // This allows us to free-ride off of the summary-details polyfill this nhs component provides.
+    .nhsuk-expander,
+    .nhsuk-expander[open] {
+        margin-bottom: 0;
+
+        .nhsuk-details__summary {
+            padding: 0;
+            border-width: 2px;
+            border-color: black;
+            border-style: solid;
+            border-radius: 4px;
+            background-color: $dull-red;
+        }
+
+        .nhsuk-details__summary-text {
+            margin-left: 16px;
+            color: black;
+            text-decoration: none;
+            // Taken from nhs expander, but with colour swapped for phe-red
+            background: url("data:image/svg+xml,%3Csvg class='nhsuk-icon nhsuk-icon__plus' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='32' aria-hidden='true'%3E%3Ccircle cx='12' cy='12' r='10' fill='%237f0029'%3E%3C/circle%3E%3Cpath fill='none' stroke='%23fff' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M12 8v8M8 12h8'%3E%3C/path%3E%3C/svg%3E%0A") left -2px center no-repeat;
+        }
+
+        &[open] .nhsuk-details__summary-text {

```

Comment:
I believe this would transpile to `.nhsuk-expander[open], .nhsuk-expander[open][open]` which will obviously work but maybe isn't ideal
Is the inclusion of .nhsuk-expander[open] in the enclosing scope a remnant of a previous version of this styling?

Reply:
Ah, very good point.
It is, yes, as there are a bunch of things that i'm overriding in both closed and open versions. I'll move this rule outside!